### ISSUE TITLE
Meteor Angular, Ionic and Cordova, apps open to white screen

### ISSUE THREAD

**Issue by sferoze** [Urigo/angular-meteor#1138]
I am creating a new app with meteor angular, ionic and cordova. 

I added platform iOS.

The app works perfectly fine in the browser

But when I do `meteor run ios` the iOS simulator opens to a white screen. 

I inspected the dom in the simulator and noticed that angular has not bootstrapped. 

I am using this code to bootstrap:

```
function onReady() {
  angular.bootstrap(document, ['ConnectedCar']);
  console.log('meteor is cordova');
}

if(Meteor.isCordova) {
  angular.element(document).on('deviceready', onReady);
} else {
  angular.element(document).ready(onReady);
}
```

Notice the log statement in the onReady function. It is run, so the angular bootstrap code is running. But why is angular not working? 

In my mobile-config.js I have `App.accessRule('*');`

In the console I do not see any errors that would signify angular not bootstrapping. 

The only thing I notice is sometimes I get 

```
WebSocket connection to 'ws://localhost:3000/sockjs/297/cku0xuqw/websocket' failed: WebSocket is closed before the connection is established.
```

Any idea why the app works perfectly fine in the browser, but not in the iOS simulator?

**Comment by ThomsC** [comment]
where have you put your `fonction onReady` it should be in `client/lib/app.js'

try to remove mobile-config.js and see what os going on you don't need it to run the simulator

`WebSocket connection to 'ws://localhost:3000/sockjs/297/cku0xuqw/websocket' failed: WebSocket is closed before the connection is established.`this happens sometimes when you don't have any subscriptions running it's not an error.

**Comment by sferoze** [comment]
@ThomsC Hi, yes the onReady is in client/lib/app.js

Still it does not seem to bootstrap the app. What else could I try?

**Comment by jessonjoy** [comment]
same issue here. runs on browser but not in ios device... onReady doesnt get called on angular bootstrap

**Comment by Urigo** [comment]
@sferoze can you share a reproduction?
also, what machine are you running on?

**Comment by matthewyt** [comment]
I am receiving the same issue. I upgraded to the Meteor 1.3 Cordova Beta and now receive the issue above. In my case it does fire onReady(), but it just loads to a blank screen so it looks like there is an issue with angular.bootstrap(document, ['']);

**Comment by sferoze** [comment]
Hi All,

The issue is not occurring anymore. To be honest I am not 100% sure what I did to fix the issue. I was just cleaning out unnecessary code and removed any packages I did not need.

I also specified in the package to use angular 1.3.0. I actually think this might have done it. Before I did not specify an angular version in the package. In the `api.use` in `package.js` file I specify angular like so `angular@1.3.0` 

I would try this if you are having an issue. I also noticed that by default Meteor selected the oldest version of some available packages. I am not sure why it did this. But I went through and specified the latest versions of these packages in `package.js `
